### DISCUSSION TITLE
Make When Moving The Map Thru A Border, Have Settings Choice to See A Pop Up Similar To the "$Country, Download" Stating the "Centered" "Country"

### DISCUSSION THREAD

**Discussion by Ere-the-Weekend-comes** [organicmaps/organicmaps#8873] [Ideas]
Hard to make a title for this

As I mentioned in the adjacent post, I was thinking of many things reading thru old topics, so I wanted to jump right in after going back thru the 1-8 pages of searches, searching (ctrl-f page search) for "Border".

This first Idea, is one of the things that brought me to Organicmaps this morning:

I was only able to tell which downloaded file I was seeing (or region, such as Western Slovania <-border-> Eastern Slovania) that I was looking at if I didn't already have it downloaded, because of the pop-up to download it.  

I was thinking how great would it be if I could see the name of the region, (the name of the file) in a popup just like this one that comes up to download, as I cross the border (then without the word download of course).  Literally the exact same popup, in the exact same style, at the exact same times of moving across a border with the center point of the screen.  Should be easy to include that since it's already there for undownloaded regions, just to have it appear just the same for downloaded regions so I always know where I am even though it is already downloaded.  

I have to say it's great that you made the words larger than the Maps.me app does, that is a major issue, the way the words there are practically illegible without a screenshot.  I can read them here at least.  But I still generally don't know which country I am looking at at any time.  I only so far downloaded 2 files for 2 regions, just to test these things, but I can no longer distinguish when I am moving between the 2.  Would be really great.  Please.  Thanks.

**Comment by Ere-the-Weekend-comes** [comment]
It could be conceivably useful also to have a setting to choose to place the name for the centered country {bold, variable color, variable transparency) always shown at the upper right or lower left or any corner user setting choice, maybe a settings choice which corner to place the country / county names.

**Comment by biodranik** [comment]
Would coloring downloaded/outdated regions #4360 be helpful?

**Reply by Ere-the-Weekend-comes** [reply]
Is that something that exists already or what?  I would perfer to see the name of the country and region or downloaded file in a corner.  I like to look at the maps and move around thru them with detail.  So I never really know where it's at without zooming out first.
